src/js/components/MaskedInput/doc.js

```diff
@@ -50,6 +50,16 @@ export const doc = MaskedInput => {
 };
 
 export const themeDoc = {
+  'maskedInput.extend': {
+    description: 'Any additional style for MaskedInput.',
+    type: 'string | (props) => {}',
+    defaultValue: undefined,
+  },
+  'text.medium': {
+    description: 'The size of the text for MaskedInput..',
```

Comment:
let's remove the extra dot at the end of the line, run `yarn test-update` and this PR will be ready to go.